src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,91 @@
+.row
+  .col
+    %p
+      During this period
+      = render partial: 'pulse_list_commits'
+    %p
+      = render partial: 'pulse_list_builds'
+      = render partial: 'pulse_list_new'
+      = render partial: 'pulse_list_changes'
+    %p
+      = render partial: 'pulse_list_branches'
+      = render partial: 'pulse_list_comments'
+
+  .col
+    .card
+      .card-header
+        Requests
+      .card-body
+        - if @requests.any?
+          .row
+            .col
+              .progress
+                -# haml-lint:disable InlineStyles
+                - @requests_by_percentage.each do |state, percentage|
+                  %div{ title: "#{@requests_by_state[state]} #{state} requests",
+                        class: "progress-bar progress-state-#{state}",
```

Comment:
The class `progress-bar` should be defined directly on the tag, so `%div.progress-bar`

Reply:
Can't do both, options hash and direct classes...

Reply:
Oh really, good to know.

Reply:
you can, but we don't do it... we always write direct classes except if interpolation is needed. It is easier to read.